### DISCUSSION TITLE
Deleting a kafka topic from the operator without deleting it from the cluster.

### DISCUSSION THREAD

**Discussion by ostecke** [strimzi/strimzi-kafka-operator#8788] [General]
I'm stuck in the following situation, perhaps somebody can provide me with some experience. 

I created some Kafka topics on a Kafka cluster which where picked up by the topic operator. Now I want to move 'ownership' of the configuration of those topics to a solution using Gitops, without losing the data inside the topics. Creating the topics in Gitops results in errors in the kafka operator of the following kind: 

```
Topic 'ABC' is already managed via KafkaTopic 'abc---e3641c36628a11df6b6b44fa28209f294e034a19' it cannot also be managed via the KafkaTopic 'abc'
```
To solve the issue I wanted to try the following:
1. stop the topic operator
2. delete `abc---e3641c36628a11df6b6b44fa28209f294e034a19` without deleting the topic on the cluster
3. create KafkaTopic 'abc' using Gitops
4. start the topic operator

Especially step 2 seems to be difficult. Anybody who can help me out here ?

**Comment by scholzj** [comment]
@tombentley Any ideas?

**Comment by klebediev** [comment]
Probably Unidirectional Topic Operator will fix this issue. Read [the link](https://strimzi.io/blog/2023/11/02/unidirectional-topic-operator/) for details
